debian/libjaeger.install

```diff
@@ -1,3 +1,4 @@
-#usr/lib/libopentracing.so.*
-#usr/lib/libjaegertracing.so.*
-#usr/lib/libthrift.so.*
+usr/lib/libopentracing.so.*
+usr/lib/libjaegertracing.so.*
+usr/lib/libthrift.so.*
+usr/lib/libyaml-cpp.so*
```

Comment:
why? i assume that this library is installed by a package, am i right?

Reply:
nope, we are building them ourselves, since we need specific versions of them

Reply:
this does not make sense, what if user want to install `libyaml-cpp-dev`?

Reply:
> nope, we are building them ourselves(using externalProjectAdd), since we need specific versions of them

I tried removing them, will test.